tests/general/RegexLintBearTest.py

```diff
@@ -0,0 +1,84 @@
+from os.path import abspath
+from queue import Queue
+
+from bears.general.RegexLintBear import RegexLintBear
+from coalib.testing.LocalBearTestHelper import (
+    LocalBearTestHelper, execute_bear)
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+
+
+test_good_python_file = """
+some_regex = r'[a-zA-Z]]'
+"""
+
+test_bad_python_file = """
+some_regex = r'(else|elseif)'
+"""
+
+test_good_cpp_file = """
+char some_regex[] = "[a-zA-Z]]";
+"""
+
+test_bad_cpp_file = """
+char some_regex[13] = "(else|elseif)";
+"""
+
+test_re_error_file = """
+some_regex = r'*ab' # This should be skipped
+some_other_regex = r'[a-z]'
+"""
+
+BAD_MESSAGE = ('E105:argv:root:0: Potential out of order alternation between '
+               '\'else\' and \'elseif\'\n'
+               '  \'(else|elseif)\'\n'
+               '         ^ here\n')
+
+
+class RegexLintBearTest(LocalBearTestHelper):
```

Comment:
this class looks like it could be implemented using `verify_local_bear` or similar wrapper.  If not, why not.

If not, please at least use `self.check_[in]validity`

Reply:
Is there any specific reason to use those? I mean through this we can asset the line number, column number, message, etc.